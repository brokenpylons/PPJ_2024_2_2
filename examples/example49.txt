(3.7 * (a + b) / (c^(1/4))) * (d * e^(1.5 * f) / g)^0.6
