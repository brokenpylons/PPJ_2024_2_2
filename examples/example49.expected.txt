lparen("(") real("3.7") times("*") lparen("(") variable("a") plus("+") variable("b") rparen(")") divides("/") lparen("(") variable("c") pow("^") lparen("(") real("1") divides("/") real("4") rparen(")") rparen(")") rparen(")") times("*") lparen("(") variable("d") times("*") variable("e") pow("^") lparen("(") real("1.5") times("*") variable("f") rparen(")") divides("/") variable("g") rparen(")") pow("^") real("0.6") 
